docs/concepts/ci-integration.md

```diff
@@ -36,16 +36,105 @@ A common requirement is to be able to publish to a private registry other than n
 beachball publish -n SOME_AUTH_TOKEN -r http://SOME_REGISTRY_URL/
 ```
 
-### package.json script
+## package.json script
 
 It's recommended to encapsulate any custom options in a `package.json` script:
 
+> Note: When running Beachball in CI, add the `-y` flag (or `--yes`). That skips the intractive prompts that are commonly unavailable in CI consoles.
+
 ```json
 {
   "scripts": {
-    "publish:beachball": "beachball publish -n $(npm.token)"
+    "publish:beachball": "beachball publish -y -n $(npm.token)"
   }
 }
 ```
 
 Then inside the CI script, simply call `yarn publish:beachball` or `npm run publish:beachball`.
+
+
+## Examples
+
+### With Azure DevOps
+
+**1 Npm authentication**. Azure DevOps has `npmAuthenticate@0` available to create a temporary npm auth token for publishing.
+
+```yml
+  - task: npmAuthenticate@0
+    inputs:
+      workingFile: .npmrc
+```
+
+**2 Git authentication**.
+
+**2.1 Add env variables.** First, you need to add environment variables. One way to do it is through a _Variable group_. Find it in Library → “+ Variable group” → “+ Add”.
+
+Add 4 env vars:
+- git.pat
+- git.user
+- git.name
+- git.email
+
+Give your pipeline permissins to access these variables in _Pipeline permissions_.
+
+**2.1 Configure Git.** You can set up your Azure DevOps pipeline to add git credentials with Git CLI or you can extrapolate it in a Node script like this:
+
+```js
+const shell = require('shelljs');
```

Comment:
Why don't we give example in terms of the yaml pipeline itself? shelling out to a task in `just-scripts` seems overkill?

I'm thinking something like this:
```yaml
steps:
  - script: git config .... 
```

Reply:
You have most of it down below in the yaml example

Reply:
👌👌 Good point! I was kind of on a fence on this one. Ideally, I would want ADO to have a pre-built task for this.